react-redux/react-redux-tests.tsx

```diff
@@ -1,7 +1,7 @@
 /// <reference path="react-redux.d.ts" />
 /// <reference path="../react/react.d.ts"/>
 /// <reference path="../react/react-dom.d.ts"/>
-/// <reference path="../redux/redux.d.ts" />
+/// <reference path="../redux/redux-1.0.0.d.ts" />
```

Comment:
I think we can use latest redux definitions. Do you have something special reason?